ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -622,3 +622,47 @@ def LinSpace(*, start, stop, num, name=None):
 
 
 Roll = to_ivy_arrays_and_back(map_raw_ops_alias(tf_frontend.roll))
+
+
+@to_ivy_arrays_and_back
+def Conv2D(
+    *,
+    input,
+    filter,
+    strides,
+    padding,
+    explicit_paddings=[],
+    use_cudnn_on_gpu=False,
+    data_format='NHWC',
+    dilations=[1, 1, 1, 1],
+    name="Conv2D",
+):
+    # ivy.backends.tensorflow expects strides and dilations to be
+    # a single integer value or a list of 2 values whereas the raw op
+    # expects a list of 4 values
+    if data_format == "NHWC":
+        strides = strides[1:-1]
+        dilations = dilations[1:-1]
+    elif data_format == "NCHW":
+        strides = strides[2:]
+        dilations = dilations[2:]
+    # for Conv2D, the explicit_padding is defined as
+    # [[0, 0], [pad_top, pad_bottom], [pad_left, pad_right], [0, 0]]
+    # when the data_format is "NHWC"
+    # for Conv2D, the explicit_paddings is defined as
+    # [[0, 0], [0, 0], [pad_top, pad_bottom], [pad_left, pad_right],
+    # when the data_format is "NCHW"
+    if padding == "EXPLICIT" and data_format == "NHWC":
+        padding = explicit_paddings[1:3]
```

Comment:
considering `tf.raw_ops.Conv2D` only takes 1d list with 8 elements as padding, you can do `padding = [explicit_paddings[i:i+2] for i in range(2, 6, 2)]` here instead.
this will convert middle 4 elements into lists which is how ivy's function accepts it.

Reply:
sure will try that now.